edx_proctoring/tests/test_student_view.py

```diff
@@ -753,7 +753,7 @@ def test_get_studentview_submitted_status_practiceexam(self):
         exam_attempt.save()
 
         rendered_response = self.render_practice_exam()
-        self.assertIn(self.practice_exam_submitted_msg, rendered_response)
+        self.assertIn(self.onboarding_exam_submitted_msg, rendered_response)
```

Comment:
Would it make sense to update the names of the functions now that the onboarding exam, and not the practice exam, is supported by the test backend? Or perhaps since the onboarding changes are temporary, it isn't worth the flip-flop. What do you think?

Reply:
Yeah, I'd stick closer to leaving all the changes we make sit behind some check of the backend's onboarding flag, so we can easily know what can be safely rolled back when we remove it